include/grpcpp/server.h

```diff
@@ -200,8 +202,9 @@ class Server final : public ServerInterface, private GrpcLibraryCodegen {
   /// the \a sync_server_cqs)
   std::vector<std::unique_ptr<SyncRequestThreadManager>> sync_req_mgrs_;
 
-  // Sever status
+  // Server status
   std::mutex mu_;
+  // Server status
```

Comment:
Since I'm requesting changes again, can I ask you to dedupe this comment?

Reply:
my bad. done.